Gems/AtomTressFX/Code/Rendering/HairCommon.cpp

```diff
@@ -39,7 +39,7 @@ namespace AZ
             Data::Instance<RPI::ShaderResourceGroup> UtilityClass::CreateShaderResourceGroup(
                 Data::Instance<RPI::Shader> shader,
                 const char* shaderResourceGroupId,
-                const char* moduleName)
+                [[maybe_unused]] const char* moduleName)
```

Comment:
Personal nit, I think it's more readable to put the `[[maybe_unused]]` at the end of the declaration - `const char* moduleName [[maybe_unused]]`